README.md

```diff
@@ -1,2 +1,3 @@
 # helena-chat
 Helena's practice task - Chat client and server
+-- test
```

Comment:
Name your branches according to the features you are doing, e.g. implement-chat-rooms. If you had Jira issues you could name them issue-1234 for example